test/test-lib/platform.ts

```diff
@@ -0,0 +1,12 @@
+import * as _ from 'lodash';
+export = () => {
+	const platform = require('../../src/platform');
```

Comment:
Can you make these typed, either via `import()` or
```ts
import * as _platform from '../../src/platform'
...
const platform: typeof _platform = require('../../src/platform');
```
I think `import()` is nicer but it does turn every call to this into a promise, but that should be fine with await/async

Reply:
```
import * as _platform from '../../src/platform';
import * as _permissions from '../../src/platform/permissions';

export const platform: typeof _platform = require('../../src/platform');
export const permissions: typeof _permissions = require('../../src/platform/permissions');
```